components/crossplane-control-plane/production/testplatform-provider-config.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: kubernetes.crossplane.io/v1alpha1
+kind: ProviderConfig
+metadata:
+  name: testplatform-kubernetes-provider-config
+  annotations:
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+spec:
+  credentials:
+    source: Secret
+    secretRef:
+      namespace: crossplane-system
+      name: testplatform-cluster
+      key: kubeconfig
+---
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: testplatform-cluster
+  namespace: crossplane-system
+  annotations:
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+    argocd.argoproj.io/sync-wave: "-1"
+spec:
+  dataFrom:
+    - extract:
+        key: production/openshift-ci/ephemeral-cluster
```

Comment:
It seems odd for this key to have `ephemeral` in the name. Is it expected to be short lived?

Reply:
`ephemeral-cluster` relates to the work I'm carrying on. The `kubeconfig` I'm extracting here is related to our main cluster `appci`. I can rename it into `production/openshift-ci/appci` if this sounds better to you.

Reply:
Yeah, or `appci-cluster` just to be extra clear.